[ui][android] Fix ModalBottomSheet expand race with Compose handler binding (#46367)
expo/expo: docs/pages/versions/unversioned/sdk/ui/jetpack-compose/bottomsheet.mdx

}
```

### Initial fully expanded state

> When `initialFullyExpanded` is `true`, the sheet opens directly in the fully expanded state on first composition while leaving the partial state reachable. Unlike `skipPartiallyExpanded`, the user can still drag down to partial and `partialExpand()` keeps working.

Reviewer: **[suggestion]** Two independent clauses joined by `and` typically read better as two sentences (`expo-docs-style-guide.md#voice-and-tone`). Splitting also lets the second fact (`partialExpand()` keeps working) stand on its own, which is the more useful API guarantee for a reader scanning the section.

```suggestion
When `initialFullyExpanded` is `true`, the sheet opens directly in the fully expanded state on first composition while leaving the partial state reachable. Unlike `skipPartiallyExpanded`, the user can still drag down to the partial state. The `partialExpand()` method also continues to work.
```
